src/test/java/bio/terra/service/filedata/azure/util/BlobIOTestUtility.java

```diff
@@ -93,6 +95,28 @@ public String generateContainerSasTokenWithReadAndListPermissions(String account
     return blobContainerClient.generateSas(sasSignatureValues);
   }
 
+  public String generateBlobSasTokenWithReadPermissions(String accountKey, String blobName) {
```

Comment:
It looks like the logic in this method is largely shared with AzureContainerPdao.getDestinationContainerSignedUrl().   Maybe in a future PR, I think I'd want to pull this into a regular util (rather than a testing util), so that we can share this chunk of code. Thoughts?

Reply:
Yeah, that's a good idea.  I think we'll have different types of Sas tokens that we need to support but we should def. consolidate them